library/mediawiki

```diff
@@ -1,9 +1,13 @@
 Maintainers: David Barratt <dbarratt@wikimedia.org> (@davidbarratt),
              Ryan Kaldari <rkaldari@wikimedia.org> (@kaldari)
 GitRepo: https://github.com/wikimedia/mediawiki-docker.git
-GitCommit: 76ad8cc3ddc56701522025583c126ad38335a364
+GitCommit: 659cf5aedb8f785bb400a63f9c450f458ead260c
+
+Tags: stable, latest, 1.29, 1.29.1
+Directory: stable
 
-Tags: stable, latest, 1.29, 1.29.0
+Tags: 1.29.0
+GitCommit: 76ad8cc3ddc56701522025583c126ad38335a364
 Directory: stable
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
@tianon oh! ok, yeah no reason to keep it. It is no longer supported.